### PR TITLE
feat: complete features page

### PR DESCRIPTION
**Pull request by Ravenclaw-OIer**
I've completed the features page!

### CODE REVIEW

**Comment by ouuan** [comment]
Well, the "Feature" section is not for promotion while the home page is. This section should be separated into several pages, each page introduces how to use a feature.

**Comment by Ravenclaw-OIer** [comment]
@ouuan What about renaming it to "whetting your appetite"? Just like the official Python documentation did. (And you wrote "the awesome features of ..." as the abstract which somehow misled me ...)

I think that this can be a more detailed promotion, and we can leave links to the descriptions.

**Comment by Ravenclaw-OIer** [comment]
[This page](https://doc.qt.io/qt-5/qt5-intro.html) also works like this one.

File: content/en/docs/Features/_index.md [hunk 1]
```diff
@@ -2,44 +2,20 @@
...
 ---
-CP Editor has a great number of awesome features that made competitive programming life easy. Here, we will show you some of the most fasicinating ones, but there are still a lot of great features for you to explore!
+This page serves as the index page of the documentation of CP Editor's features
 
-## Editor to your preference
-CP Editor is highly customizable, from global theme to editor theme, from compiler flags to run arguments, from key bindings to time limits, all aspects of your CP life are decided by you.
+*To developers and contributors: Do not forget to update this page after completing another part of documentation!*
 
-We know that a beautiful editor theme will surely boost your performance, so, we provide 5 different themes out of the box. If your preferred theme is not included, you are more than welcomed to file a feature request [here](https://github.com/cpeditor/cpeditor/issues). You can also produce your own theme as CP Editor is free software and its code can be read, modified and redistributed by everyone. We would be delighted if you can contribute your theme to the project!
+## Documentation Index
```
Comment (ouuan) [inline content/en/docs/Features/_index.md:11]: The index is generated by the Docsy theme (or, by Hugo), not manually.

File: content/en/docs/Features/_index.md [hunk 2]
```diff
@@ -2,44 +2,20 @@
 title: "Features"
 linkTitle: "Features"
 weight: 40
-description: The awesome features of CP Editor
+description: Find how-to-s for features hear
```
Comment (ouuan) [inline content/en/docs/Features/_index.md:5]: The documentation is "how-to", you don't need to emphasize it.

**Comment by ouuan** [comment]
You can open another PR for it. Just leave the feature page unchanged.

**Pull request closed without merging**